modules/ppcp-button/resources/js/modules/Renderer/CreditCardRenderer.js

```diff
@@ -218,7 +218,8 @@ class CreditCardRenderer {
             }).catch(err => {
                 this.spinner.unblock();
                 this.errorHandler.clear();
-                if (err.details.length > 0) {
+
+                if (err.hasOwnProperty('details')) {
```

Comment:
hm, I think `hasOwnProperty` is not intended for such purposes.

Maybe just `if (err.details)`?

Also you can check `length` like this: `err.details?.length > 0`

Reply:
yup, much better with `err.details`, thanks.